src/helpers/performance-report-parser.helper.js

```diff
@@ -0,0 +1,30 @@
+import fs from 'fs';
+import chalk from 'chalk';
+
+class PerformanceReportParse {
```

Comment:
typo: missing "r" at the end

Reply:
Changed helper name to `PerformanceReportAnalyser`